src/components/Routes/Routes.tsx

```diff
@@ -31,6 +36,10 @@ export function Routes() {
   const location = useLocation();
 
   const isJuju = useSelector(getIsJuju);
+  const wsControllerURL = useSelector(getWSControllerURL);
+  const crossModelQueriesSupported = useAppSelector((state) =>
+    getControllerFeatureEnabled(state, wsControllerURL, "crossModelQueries")
+  );
```

Comment:
Was wondering if maybe it might make sense to make this its own selector, as we are using this exact selector in 2 places.

Reply:
Good idea. I've added a selector that also retrieves the other data it needs, so this simplifies the components quite a bit.